scenarios/writing_clamp_function_roc.yaml

```diff
@@ -1,97 +1,88 @@
 first: '1'
 screens:
   1: |
-    ┃
+    ┃🗌
   2: |
     clamp : 🗌
     clamp┃ = 🗌
   3: |
     clamp : 🗌
-    clamp = ┃
+    clamp = ┃🗌
   4: |
     clamp : 🗌 -> 🗌
-    clamp = \┃ ->
+    clamp = \┃🗌 ->
       🗌
   5: |
     clamp : 🗌 -> 🗌
     clamp = \v┃ ->
       🗌
   6: |
-      clamp : 🗌, 🗌 -> 🗌
-      clamp = \v, ┃ ->
-        🗌
+    clamp : 🗌, 🗌 -> 🗌
+    clamp = \v, ┃🗌 ->
+      🗌
   7: |
     clamp : 🗌, 🗌 -> 🗌
     clamp = \v, low┃ ->
       🗌
   8: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, ┃ ->
+    clamp = \v, low, ┃🗌 ->
       🗌
   9: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high┃ ->
       🗌
   10: |
     clamp : 🗌, 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, high, ┃ ->
+    clamp = \v, low, high, ┃🗌 ->
       🗌
   11: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high ->
-      ┃
+      ┃🗌
   12: |
-    clamp : a, *, * -> a
+    clamp : a, 🗌, 🗌 -> a
```

Comment:
I don't think Roc supports a standalone star, a Blank is basically the same thing.
Roc does also not support a Blank right now, but the editor does :)